DataFormats/Headers/include/Headers/DataHeader.h

```diff
@@ -711,10 +712,15 @@ struct DataHeader : public BaseHeader {
   //___NEW STUFF GOES BELOW
 
   ///
-  /// first orbit of time frame as unique identifier within the run
+  /// first orbit of time frame as unique identifier within the run (unless the TF is replayed)
   ///
   TForbitType firstTForbit;
 
+  ///
```

Comment:
can you please add a comment here stating this was added in Version3. I need to convert old recorded data to new DH versions, and looking this up via git is not ideal. (and we don't keep the old structure definitions around, like for the raw header) Thanks